promscale/installation/kubernetes/tobs.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Install the observability stack (tobs) for Kubernetes
+excerpt: Install tobs
+product: promscale
+keywords: [analytics, Kubernetes]
+tags: [install, k8s]
+related_pages:
+  - /promscale/:currentVersion:/installation/resource-recomm
+  - /promscale/:currentVersion:/send-data/
+---
+
+import PromscaleConfigurationRecommendations from 'versionContent/_partials/_promscale-configuration-recommendations.mdx';
+
+# Install the observability stack (tobs) for Kubernetes
+The Observability Stack, is a pre-packaged distribution of observability tools
+and dashboard interfaces which can be installed on any existing Kubernetes
+cluster. It includes many of the most popular open-source observability tools
+such as Prometheus, Grafana, Jaeager, Promlens, TimescaleDB, and others.
+Together, these provide a maintainable solution to analyze the traffic on the
+server and identify any potential problems with a deployment. You can use Helm
+charts to configure and update [tobs][tobs] deployments. 
+
+## Before you begin
+*  Understand the [resource requirements][resource-requirements] for your cluster.
+*  [Deploy a Kubernetes Cluster][kubernetes-cluster]. 
+*  Install [kubectl][kubectl] in your local environment and
+   [connect to your cluster][connect-to-cluster].
+*  Install [Helm 3][helm] in your local machine to add and update the repositories:
+   1.  Add the stable Helm charts repository to your Helm repositories.
```

Comment:
I would be tempted to move this out of the bullet list. Maybe just start a new para and say "If you need to install Helm on your local machine, use these commands:" or similar.